README.md

```diff
@@ -1,99 +1,63 @@
-Poseidon is Firmament's (http://www.firmament.io) integration with
-Kubernetes.
+# Introduction
+The Poseidon/Firmament scheduler incubation project is to bring integration of Firmament Scheduler (OSDI paper) in Kubernetes.
+At a very high level, Poseidon/Firmament scheduler augments the 
+current Kubernetes scheduling capabilities by incorporating a new 
+novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. 
+Firmament models workloads on a cluster as flow networks and runs min-cost flow optimizations over these networks to make scheduling decisions.
 
-[![Build Status](https://travis-ci.org/camsas/poseidon.svg)](https://travis-ci.org/camsas/poseidon)
-[![Coverage Status](https://coveralls.io/repos/github/camsas/poseidon/badge.svg?branch=master)](https://coveralls.io/github/camsas/poseidon?branch=master)
+Due to the inherent rescheduling capabilities, the new scheduler enables a globally optimal scheduling environment that keeps on refining the dynamic placements of the workload.
 
-***Note: this repo contains an initial prototype, it may break at any time! :)***
+As we all know that as part of the Kubernetes multiple schedulers support, each new pod is typically scheduled by the default scheduler, but Kubernetes can be instructed to use another scheduler by specifying the name of another custom scheduler (Poseidon in our case) at the time of pod deployment. In this case, the default scheduler will ignore that Pod and allow Poseidon scheduler to schedule the Pod on a relevant node. We plugin Poseidon as an add-on scheduler to K8s, by using the 'schedulerName' as Poseidon in the pod template this will by-pass the default-scheduler.
 
-# Getting started
+# Key Advantages
 
-The easiest way to get Poseidon up and running is to use our Docker image:
+* Flow graph scheduling provides the following 
+  * Support for high-volume workloads placement 
+  * Complex rule constraints 
+  * Globally optimal scheduling
```

Comment:
Again, this isn't true -- it's only globally optimal for a given policy on the Firmament side.

Reply:
Yeah agree, Shiva can you instead word this sentence as "Globally optimal scheduling (for a given scheduling policy)".